modules/mod_ginger_embed/dispatch/dispatch

```diff
@@ -1,4 +1,4 @@
 [
-    {embed_js, ["embed", "embed.js"], controller_file, [{path, "js/embed.js"}, {root, [lib]}]},
+    {embed, ["gingerembed", id], controller_template, [{template, "embed.tpl"}]},
```

Comment:
Let’s not unnecessarily change the URL. The old one was fine (`["embed", id]`).

Reply:
agreed

Reply:
Done.

Reply:
Agreed, there might have been some reasoning about it at the time, not sure now. If this is reverted this https://github.com/driebit/ginger/blob/95fa7da2f2bd299745f82402b1341f560659209f/modules/mod_ginger_embed/mod_ginger_embed.erl#L66 has to be modified as well.

Reply:
shit! no we need ginger embed, because of the whitelisting

Reply:
No, just change the above snippet to`/embedd`, too.

Reply:
ok